IBAnimatable/Animatable.swift

```diff
@@ -22,12 +22,12 @@ public protocol Animatable: class {
   /**
    Animation duration (in seconds)
    */
-  var duration: Double { get set }
+  var duration: TimeInterval { get set }
```

Comment:
I tried, but that creates SwiftLint warnings since we have to use an explicit type (instead of typealias) for the inspectable properties :(